matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -385,7 +385,7 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
                 }
 
         val closedTime = existingPollSummary?.closedTime
-        if (closedTime != null && eventTimestamp > closedTime) {
+        if (closedTime != null) {
```

Comment:
is there a case where the poll could have a closed timestamp but still be valid to casting votes? cc @onurays

Reply:
@onurays bump

Reply:
I think we should keep this condition. Even if we prevent it previously, there is always a chance to trigger a poll response to an ended poll from another client or via rest api.

Reply:
Reverted this change and pushed alternative solution. See below comment for details